src/plugins/data_source_management/public/components/create_button/create_button.tsx

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+import * as H from 'history';
```

Comment:
not sure if import * is a good practice, I think we will use mostly likely just History? :P

Reply:
Refactored